database/liquibase/changelogs/schemas/init_schemas.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>   
+<databaseChangeLog
+   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd
+      http://www.liquibase.org/xml/ns/pro 
+      http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd">
+   <changeSet author="stankowd" id="1">
+      <sql dbms="postgresql" endDelimiter=";">
+        CREATE SCHEMA events
+      </sql>
+      <sql dbms="postgresql" endDelimiter=";">
+        CREATE SCHEMA users
+      </sql>
+      <sql dbms="postgresql" endDelimiter=";">
+        CREATE SCHEMA games
+      </sql>
+      <sql dbms="postgresql" endDelimiter=";">
+        CREATE SCHEMA payments
+      </sql>
+   </changeSet>
```

Comment:
Might be more of an @LARP2023Z/architects' decision, but I think separate db's was the goal instead of one DB with schemas

Reply:
True, we want to have one database for each service